classes/game.rb

```diff
@@ -12,9 +12,16 @@ def initialize(publish_date, last_played_at, multiplayer)
   private
 
   def can_be_archived?
-    return false unless super
-
-    last_played_years = (Time.now - @last_played_at).to_i / (365 * 24 * 60 * 60)
-    last_played_years > 2
+    last_played_year = @last_played_at.year
+    current_year = Time.now.year
+    years_since_last_played = current_year - last_played_year
+    years_since_last_played > 2
   end
+
+  # def can_be_archived?
+  #   return false unless super
+
+  #   last_played_years = (Time.now - @last_played_at).to_i / (365 * 24 * 60 * 60)
+  #   last_played_years > 2
+  # end
```

Comment:
It would be best to remove the commented code from your codebase to follow best practices and improve the code readability.